client/src/components/context/receipt/ReceiptState.js

```diff
@@ -10,28 +10,37 @@ import {
   GET_ALL_RECEIPTS,
   GET_RECEIPTS_YEARLY,
   CLEAR_RECEIPT,
-} from '../actionTypes';
+} from "../actionTypes";
 
 const ReceiptState = (props) => {
   const initialState = {
     loading: true,
     receipts: [],
-    statusMessage: '',
-    errorMessage: '',
-    activeTab: 'dashboard',
-    totalExpense: '',
+    statusMessage: "",
+    errorMessage: "",
+    activeTab: "dashboard",
+    totalExpense: "",
   };
   const [state, dispatch] = useReducer(receiptReducer, initialState);
 
   // Create receipt
   const createReceipt = async (receiptData) => {
     const config = {
       header: {
-        'Content-Type': 'application/json',
+        "Content-Type": "application/json",
       },
     };
     try {
-      const res = await axios.post('/createReceipt', receiptData, config);
+      console.log(receiptData);
+      const formData = new FormData();
+      const file = receiptData.picture_url[0];
+
+      console.log(file.name, file.data);
+
+      formData.append("images", file);
```

Comment:
I'm just grabbing the first file uploaded - but you can also just send in the entire picture_url array no problem

Reply:
you'll have to append the other data as well 

formData.append('amount', amount) 

etc